cas-server-support-osf/src/main/java/io/cos/cas/adaptors/postgres/models/OpenScienceFrameworkTimeBasedOneTimePassword.java

```diff
@@ -50,8 +55,9 @@
     @Column(name = "is_confirmed", nullable = false)
     private Boolean confirmed;
 
-    @Column(name = "is_deleted", nullable = false)
-    private Boolean deleted;
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column(name = "deleted", nullable = false)
```

Comment:
As mentioned by @felliott , `deleted` needs to be `nullable` by definition. This doesn't break CAS since CAS is read-only when access OSF DB.